docs/src/main/asciidoc/security-jwt.adoc

```diff
@@ -975,9 +974,12 @@ Note that you can't access the injected `JsonWebToken` in the public methods if
 [[add-smallrye-jwt]]
 === How to Add SmallRye JWT directly
 
-If you work with Quarkus extensions which do not support `HTTP` (for example, `Quarkus GRPC`) or provide their own extension specific `HTTP` support conflicting with the one offered by `quarkus-smallrye-jwt` and `Vert.x HTTP` (example, `Quarkus Amazon Lambda`) and you would like to <<jwt-parser, Parse and Verify JsonWebToken with JWTParser>> then please use `smallrye-jwt` directly instead of `quarkus-smallrye-jwt`.
+To xref:jwt-parser[Parse and Verify JsonWebToken with JWTParser], use `smallrye-jwt` instead of `quarkus-smallrye-jwt` directly for the following situations:
```

Comment:
Same comment about using  (and declaring) ID [[ ]]

Reply:
**_jwt-parser_** is that ID. This link is fine